c/src/ml-api-service.c

```diff
@@ -297,8 +300,22 @@ _ml_service_offloading_conf_to_opt (JsonObject * object, const gchar * name,
   for (iter = list; iter != NULL; iter = g_list_next (iter)) {
     const gchar *key = iter->data;
     const gchar *val = json_object_get_string_member (offloading_object, key);
-
     status = ml_option_set (option, key, g_strdup (val), g_free);
+
+    if (g_ascii_strcasecmp (iter->data, "training") == 0) {
```

Comment:
Let's use `key` instead of `iter->data`. L305, L317, L318

Reply:
okay, I will apply it to the last pr.